elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -256,21 +256,21 @@ $variables: (
   --pfe-cta--BackgroundColor: transparent;
   --pfe-cta--BorderColor: var(
     --pfe-broadcasted--text,
-    #{pfe-color(ui-complement)}
+    #{pfe-color(ui-base)}
```

Comment:
This will be a difficult merge with the local variables branch.  Just a heads up.

Reply:
👍